paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptor.kt

```diff
@@ -177,8 +177,12 @@ internal class DefaultIntentConfirmationInterceptor @Inject constructor(
         shippingValues: ConfirmPaymentIntentParams.Shipping?,
         setupForFutureUsage: ConfirmPaymentIntentParams.SetupFutureUsage?,
     ): NextStep {
+        var attribution = paymentMethodCreateParams.attribution + "deferred-intent"
```

Comment:
Nit: Using `buildSet { … }` would be a little cleaner, especially for the conditional addition of `autopm`.